<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSandOptions</class>
 <widget class="QWidget" name="WdgSandOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>210</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>11</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="radiusLayout">
      <item>
       <widget class="QLabel" name="radiusLabel">
        <property name="text">
         <string>radius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="radiusSpinBox">
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="radiusSlider">
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="amountLayout">
      <item>
       <widget class="QLabel" name="amountLabel">
        <property name="text">
         <string>amount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="amountSpinBox">
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="amountSlider">
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="sandDepletionCHBox">
      <property name="text">
       <string>Sand depletion</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amountSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>amountSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amountSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>amountSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
